custom/icds_reports/dashboard_utils.py

```diff
@@ -60,3 +61,208 @@ def _get_nav_metadatada():
             'image': static('icds_reports/mobile/images/bulb.png')
         },
     }
+
+
+def _get_nav_menu_items():
+    return {
```

Comment:
Its looks entirely a json. It may make sense to put it into a config json(may be called as UI_routes_config.json) file instead of doing it in python

Reply:
I can see reasons to do this in JS and in Python. I think you're right that JS might be a better fit for this, but we can address more holistically via https://trello.com/c/3wjmEekU/217-move-the-config-dict-into-js